### PR TITLE
Fix/translation

### PR DESCRIPTION
**Pull request by sushil-truemark**
TASK:
- updated translation

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -14,15 +14,16 @@ en:
...
+    spree_slider_locations:
+      config_name: 'Spree Slider Locations'
+      
   activerecord:
     models:
       spree/slide: 'Slide'
 
   spree_slider:
-    config_name: 'Spree Slider'
     config_description: 'Manage Spree Slider contents'
```
Comment (poudelprakash) [inline config/locales/en.yml]: can we pull of this change without removing these changes..it is hard to follow up what is happening here?

File: config/initializers/spree.rb [hunk 2]
```diff
@@ -17,8 +17,8 @@
       .with_admin_ability_check(Spree::Slide)
```
Comment (poudelprakash) [inline config/initializers/spree.rb]: @sushil-truemark seems like you are skipping ability check

Comment (poudelprakash) [reply config/initializers/spree.rb]: I think you are recreating the function from line 14..and we should consolidate changes instead of just adding new configs..it will leave dead code and create confusion

Comment (sushil-truemark) [reply config/initializers/spree.rb]: Unfortunately, I am not able to understand how we are recreating the function here and skipping the ability check. 
This applied way is one suggested by [spree documentation](https://dev-docs.spreecommerce.org/customization/admin-panel) and default menu items are creating using similar way. 

https://github.com/spree/spree_backend/blob/main/app/models/spree/admin/main_menu/default_configuration_builder.rb#L80



image

Comment (sushil-truemark) [reply config/initializers/spree.rb]: Also, Deface is no longer recommended way.
https://dev-docs.spreecommerce.org/advanced/deface_overrides_tutorial
image

Comment (poudelprakash) [reply config/initializers/spree.rb]: @sushil-truemark I am just reffering to line 17 I commented on..
it was checking ability previously in initializer

**Comment by poudelprakash** [comment]
I am merging this, @sushil-truemark please create an issue to migrate all the translations into new standard with all the required details minus sensitive info..
